src/main/java/com/revature/domain/Batch.java

```diff
@@ -15,26 +13,24 @@
 import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-
 @Entity
 @Table(name="ie_batch")
 @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
-public class Batch implements Serializable  {
+public class Batch {
 
 	
 	@Id
 	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="batch_sequence")
 	@SequenceGenerator(allocationSize=1, name="batch_sequence", sequenceName="batch_seq")
 	@Column(name="b_id")
-	private int id;
+	private Integer id;
 	
 	@Column(name="b_name")
 	private String name;
 	
-	@ManyToMany(cascade=CascadeType.ALL)
+	@ManyToMany
 	@JoinTable(name="ie_person_batch", joinColumns=@JoinColumn(name="batch_id"), inverseJoinColumns=@JoinColumn(name="person_id"))
-	private List<Person> persons;
+	private Set<Person> persons;
 
 	public Batch() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)